"PYTHONNOUSERSITE=True" leaks into shell scripts run by 'sct_run_batch'

### Description

I am using `sct_run_batch` and in the shell script called, I call a command from [gradient_unwarp](https://github.com/Washington-University/gradunwarp/tree/v1.2.0). I downloaded SCT 5.1.0 and when calling this line :
~~~
gradient_unwarp.py ${file_t1}.nii.gz ${file_t1}_gradcorr.nii.gz siemens -g ${PATH_GRADCORR_FILE}/coeff.grad -n
~~~
This error occurs:
~~~
+ gradient_unwarp.py sub-1000394_T1w_raw_RPI_r.nii.gz sub-1000394_T1w_raw_RPI_r_gradcorr.nii.gz siemens -g /mnt/y/sebeda/coeff.grad/ICM/coeff.grad -n
Traceback (most recent call last):
  File "/usr/local/bin/gradient_unwarp.py", line 11, in <module>
    from gradunwarp.core import (globals, coeffs, utils)
  File "/usr/local/lib/python3.7/dist-packages/gradunwarp/core/__init__.py", line 8, in <module>
    from .unwarp_resample import Unwarper
  File "/usr/local/lib/python3.7/dist-packages/gradunwarp/core/unwarp_resample.py", line 8, in <module>
    import numpy as np
ModuleNotFoundError: No module named 'numpy'
~~~
I tried runnig the same command in my terminal, there was no error. I went back to SCT v5.0.1 and there was no error, the command worked.
I am running on WSL1 with Ubuntu 18.04.

### Steps to Reproduce
See [README.md](https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/split_processing/README.md) (preprocessing section)

0. Install SCT 5.1.0

1. Initialize shell variable with path to gradient correction file
~~~
PATH_GRADCORR_FILE=<path-gradcorr>
~~~
2. Run preprocessing (see [preprocess_data.sh](https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/blob/split_processing/preprocess_data.sh))
~~~
sct_run_batch -jobs -1 -path-data <PATH-DATA> -path-output ~/ukbiobank_preprocess -script preprocess_data.sh -script-args $PATH_GRADCORR_FILE
~~~

**Expected behavior:** Using SCT v5.0.1 or running the command in the terminal:
~~~
 gradient_unwarp.py sub-1000032_T1w_raw_RPI_r.nii.gz sub-1000032_T1w_raw_RPI_r_gradcorr.nii.gz siemens -g /mnt/y/sebeda/coeff.grad/ICM/coeff.grad -n
gradunwarp-INFO: Parsing /mnt/y/sebeda/coeff.grad/ICM/coeff.grad for harmonics coeffs
gradunwarp-INFO: Evaluating spherical harmonics
gradunwarp-INFO: on a 60^3 grid
gradunwarp-INFO: with extents -300.0mm to 300.0mm
gradunwarp-INFO: along x...
...
~~~

**Actual behavior:** Here is the complete log file of the processing error.
[err.preprocess_data_sub-1000394.log](https://github.com/neuropoly/spinalcordtoolbox/files/5875679/err.preprocess_data_sub-1000394.log)

For example installing to local directory:

```console
[drulex@gentoo-p53 tmp]  $  git clone git@github.com:Washington-University/gradunwarp.git
Cloning into 'gradunwarp'...
remote: Enumerating objects: 378, done.
remote: Total 378 (delta 0), reused 0 (delta 0), pack-reused 378
Receiving objects: 100% (378/378), 76.08 KiB | 1.77 MiB/s, done.
Resolving deltas: 100% (206/206), done.
[drulex@gentoo-p53 tmp]  $  cd gradunwarp/
[drulex@gentoo-p53 gradunwarp] [master] $  python setup.py install --prefix ./
running install
running build
running config_cc
unifing config_cc, config, build_clib, build_ext, build commands --compiler options
running config_fc
unifing config_fc, config, build_clib, build_ext, build commands --fcompiler options
running build_src
build_src
building py_modules sources
building extension "gradunwarp.core.interp3_ext" sources
building extension "gradunwarp.core.legendre_ext" sources
building extension "gradunwarp.core.transform_coordinates_ext" sources
building data_files sources
build_src: building npy-pkg config files
running build_py
package init file 'gradunwarp/core/tests/__init__.py' not found (or not a regular file)
creating build
creating build/lib.linux-x86_64-3.7
creating build/lib.linux-x86_64-3.7/gradunwarp
creating build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/core/__init__.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/core/coeffs.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/core/globals.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/__init__.py -> build/lib.linux-x86_64-3.7/gradunwarp
copying gradunwarp/core/utils.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/core/unwarp_resample.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
copying gradunwarp/core/gradient_unwarp.py -> build/lib.linux-x86_64-3.7/gradunwarp/core
creating build/lib.linux-x86_64-3.7/gradunwarp/core/tests
copying gradunwarp/core/tests/test_utils.py -> build/lib.linux-x86_64-3.7/gradunwarp/core/tests
package init file 'gradunwarp/core/tests/__init__.py' not found (or not a regular file)
running build_ext
customize UnixCCompiler
customize UnixCCompiler using build_ext
building 'gradunwarp.core.interp3_ext' extension
compiling C sources
C compiler: x86_64-pc-linux-gnu-gcc -pthread -fPIC

creating build/temp.linux-x86_64-3.7/gradunwarp
creating build/temp.linux-x86_64-3.7/gradunwarp/core
compile options: '-I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/include/python3.7m -c'
extra options: '-O3'
x86_64-pc-linux-gnu-gcc: gradunwarp/core/interp3_ext.c
In file included from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1822,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from gradunwarp/core/interp3_ext.c:10:
/usr/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
gradunwarp/core/interp3_ext.c: In function ‘interp3’:
gradunwarp/core/interp3_ext.c:97:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
   97 |     itr_v = (PyArrayIterObject *) PyArray_IterNew(result);
      |                                                   ^~~~~~
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/interp3_ext.c:97:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/interp3_ext.c:98:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
   98 |     itr_r = (PyArrayIterObject *) PyArray_IterNew(R);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/interp3_ext.c:98:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/interp3_ext.c:99:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
   99 |     itr_c = (PyArrayIterObject *) PyArray_IterNew(C);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/interp3_ext.c:99:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/interp3_ext.c:100:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  100 |     itr_s = (PyArrayIterObject *) PyArray_IterNew(S);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/interp3_ext.c:100:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/interp3_ext.c:123:9: warning: returning ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’} from a function with incompatible return type ‘PyObject *’ {aka ‘struct _object *’} [-Wincompatible-pointer-types]
  123 |  return result;
      |         ^~~~~~
x86_64-pc-linux-gnu-gcc -pthread -shared -Wl,-O1 -Wl,--as-needed -L. build/temp.linux-x86_64-3.7/gradunwarp/core/interp3_ext.o -L/usr/lib64 -lpython3.7m -o build/lib.linux-x86_64-3.7/gradunwarp/core/interp3_ext.cpython-37m-x86_64-linux-gnu.so
building 'gradunwarp.core.legendre_ext' extension
compiling C sources
C compiler: x86_64-pc-linux-gnu-gcc -pthread -fPIC

compile options: '-I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/include/python3.7m -c'
extra options: '-O3'
x86_64-pc-linux-gnu-gcc: gradunwarp/core/legendre_ext.c
In file included from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1822,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from gradunwarp/core/legendre_ext.c:10:
/usr/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
gradunwarp/core/legendre_ext.c: In function ‘legendre’:
gradunwarp/core/legendre_ext.c:86:25: warning: format ‘%d’ expects argument of type ‘int’, but argument 2 has type ‘long int’ [-Wformat=]
   86 |         printf("but mu=%d and nu=%d\n", mu, nu);
      |                        ~^               ~~
      |                         |               |
      |                         int             long int
      |                        %ld
gradunwarp/core/legendre_ext.c:86:35: warning: format ‘%d’ expects argument of type ‘int’, but argument 3 has type ‘long int’ [-Wformat=]
   86 |         printf("but mu=%d and nu=%d\n", mu, nu);
      |                                  ~^         ~~
      |                                   |         |
      |                                   int       long int
      |                                  %ld
gradunwarp/core/legendre_ext.c:99:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
   99 |     itr_x = (PyArrayIterObject *) PyArray_IterNew(x);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/legendre_ext.c:99:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/legendre_ext.c:100:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  100 |     itr_r = (PyArrayIterObject *) PyArray_IterNew(result);
      |                                                   ^~~~~~
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/legendre_ext.c:100:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/legendre_ext.c:108:32: warning: format ‘%f’ expects argument of type ‘doubl ’, but argument 2 has type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’} [-Wformat=]
  108 |             printf("but got x=%f\n", x);
      |                               ~^     ~
      |                                |     |
      |                                |     PyArrayObject * {aka struct tagPyArrayObject_fields *}
      |                                double
gradunwarp/core/legendre_ext.c:157:12: warning: returning ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’} from a function with incompatible return type ‘PyObject *’ {aka ‘struct _object *’} [-Wincompatible-pointer-types]
  157 |     return result;
      |            ^~~~~~
x86_64-pc-linux-gnu-gcc -pthread -shared -Wl,-O1 -Wl,--as-needed -L. build/temp.linux-x86_64-3.7/gradunwarp/core/legendre_ext.o -L/usr/lib64 -lpython3.7m -o build/lib.linux-x86_64-3.7/gradunwarp/core/legendre_ext.cpython-37m-x86_64-linux-gnu.so
building 'gradunwarp.core.transform_coordinates_ext' extension
compiling C sources
C compiler: x86_64-pc-linux-gnu-gcc -pthread -fPIC

compile options: '-I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/lib/python3.7/site-packages/numpy/core/include -I/usr/include/python3.7m -c'
extra options: '-O3'
x86_64-pc-linux-gnu-gcc: gradunwarp/core/transform_coordinates_ext.c
In file included from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarraytypes.h:1822,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/ndarrayobject.h:12,
                 from /usr/lib/python3.7/site-packages/numpy/core/include/numpy/arrayobject.h:4,
                 from gradunwarp/core/transform_coordinates_ext.c:10:
/usr/lib/python3.7/site-packages/numpy/core/include/numpy/npy_1_7_deprecated_api.h:17:2: warning: #warning "Using deprecated NumPy API, disable it with " "#define NPY_NO_DEPRECATED_API NPY_1_7_API_VERSION" [-Wcpp]
   17 | #warning "Using deprecated NumPy API, disable it with " \
      |  ^~~~~~~
gradunwarp/core/transform_coordinates_ext.c: In function ‘transform_coordinates’:
gradunwarp/core/transform_coordinates_ext.c:102:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  102 |     itr_x = (PyArrayIterObject *) PyArray_IterNew(X);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:102:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/transform_coordinates_ext.c:103:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  103 |     itr_y = (PyArrayIterObject *) PyArray_IterNew(Y);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:103:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/transform_coordinates_ext.c:104:51: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  104 |     itr_z = (PyArrayIterObject *) PyArray_IterNew(Z);
      |                                                   ^
      |                                                   |
      |                                                   PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:104:51: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/transform_coordinates_ext.c:105:52: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  105 |     itr_xm = (PyArrayIterObject *) PyArray_IterNew(Xm);
      |                                                    ^~
      |                                                    |
      |                                                    PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:105:52: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/transform_coordinates_ext.c:106:52: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  106 |     itr_ym = (PyArrayIterObject *) PyArray_IterNew(Ym);
      |                                                    ^~
      |                                                    |
      |                                                    PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:106:52: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
gradunwarp/core/transform_coordinates_ext.c:107:52: warning: passing argument 1 of ‘(PyObject * (*)(PyObject *))*(PyArray_API + 784)’ from incompatible pointer type [-Wincompatible-pointer-types]
  107 |     itr_zm = (PyArrayIterObject *) PyArray_IterNew(Zm);
      |                                                    ^~
      |                                                    |
      |                                                    PyArrayObject * {aka struct tagPyArrayObject_fields *}
gradunwarp/core/transform_coordinates_ext.c:107:52: note: expected ‘PyObject *’ {aka ‘struct _object *’} but argument is of type ‘PyArrayObject *’ {aka ‘struct tagPyArrayObject_fields *’}
x86_64-pc-linux-gnu-gcc -pthread -shared -Wl,-O1 -Wl,--as-needed -L. build/temp.linux-x86_64-3.7/gradunwarp/core/transform_coordinates_ext.o -L/usr/lib64 -lpython3.7m -o build/lib.linux-x86_64-3.7/gradunwarp/core/transform_coordinates_ext.cpython-37m-x86_64-linux-gnu.so
running build_scripts
creating build/scripts.linux-x86_64-3.7
copying and adjusting gradunwarp/core/gradient_unwarp.py -> build/scripts.linux-x86_64-3.7
changing mode of build/scripts.linux-x86_64-3.7/gradient_unwarp.py from 644 to 755
running install_lib
creating lib
creating lib/python3.7
creating lib/python3.7/site-packages
creating lib/python3.7/site-packages/gradunwarp
copying build/lib.linux-x86_64-3.7/gradunwarp/__init__.py -> .//lib/python3.7/site-packages/gradunwarp
creating lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/transform_coordinates_ext.cpython-37m-x86_64-linux-gnu.so -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/legendre_ext.cpython-37m-x86_64-linux-gnu.so -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/interp3_ext.cpython-37m-x86_64-linux-gnu.so -> .//lib/python3.7/site-packages/gradunwarp/core
creating lib/python3.7/site-packages/gradunwarp/core/tests
copying build/lib.linux-x86_64-3.7/gradunwarp/core/tests/test_utils.py -> .//lib/python3.7/site-packages/gradunwarp/core/tests
copying build/lib.linux-x86_64-3.7/gradunwarp/core/gradient_unwarp.py -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/unwarp_resample.py -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/utils.py -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/globals.py -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/coeffs.py -> .//lib/python3.7/site-packages/gradunwarp/core
copying build/lib.linux-x86_64-3.7/gradunwarp/core/__init__.py -> .//lib/python3.7/site-packages/gradunwarp/core
byte-compiling .//lib/python3.7/site-packages/gradunwarp/__init__.py to __init__.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/tests/test_utils.py to test_utils.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/gradient_unwarp.py to gradient_unwarp.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/unwarp_resample.py to unwarp_resample.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/utils.py to utils.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/globals.py to globals.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/coeffs.py to coeffs.cpython-37.pyc
byte-compiling .//lib/python3.7/site-packages/gradunwarp/core/__init__.py to __init__.cpython-37.pyc
running install_scripts
creating bin
copying build/scripts.linux-x86_64-3.7/gradient_unwarp.py -> .//bin
changing mode of .//bin/gradient_unwarp.py to 755
running install_data
copying gradunwarp/core/interp3_ext.c -> .//lib/python3.7/site-packages/gradunwarp/core/
copying gradunwarp/core/transform_coordinates_ext.c -> .//lib/python3.7/site-packages/gradunwarp/core/
copying gradunwarp/core/legendre_ext.c -> .//lib/python3.7/site-packages/gradunwarp/core/
running install_egg_info
Writing .//lib/python3.7/site-packages/gradunwarp-HCP_1.2.0-py3.7.egg-info
running install_clib
customize UnixCCompiler
[drulex@gentoo-p53 gradunwarp] [master] $  PYTHONPATH+=./ ./bin/gradient_unwarp.py -h
usage: 
gradient_unwarp infile outfile manufacturer -g <coefficient file> [optional arguments]

positional arguments:
  infile                The input warped file (nifti or mgh)
  outfile               The output unwarped file (extension should be
                        .nii/.nii.gz/.mgh/.mgz)
  {siemens,ge}          vendor (either "ge" or "siemens" for now)

optional arguments:
  -h, --help            show this help message and exit
  --version, -v         show program's version number and exit
  -g GRADFILE, --gradfile GRADFILE
                        The .grad coefficient file
  -c COEFFILE, --coeffile COEFFILE
                        The .coef coefficient file
  -w, --warp            warp a volume (as opposed to unwarping)
  -n, --nojacobian      Do not perform Jacobian intensity correction
  --fovmin FOVMIN       the minimum extent of harmonics evaluation grid in
                        meters
  --fovmax FOVMAX       the maximum extent of harmonics evaluation grid in
                        meters
  --numpoints NUMPOINTS
                        number of grid points in each direction
  --interp_order ORDER  the order of interpolation(1..4) where 1 is linear -
                        default
  --verbose
[drulex@gentoo-p53 gradunwarp] [master] $  
```

Of course you would need to update `PYTHONPATH` and add the local dir to `PATH`

--- a/spinalcordtoolbox/scripts/sct_run_batch.py
+++ b/spinalcordtoolbox/scripts/sct_run_batch.py
@@ -198,7 +198,7 @@ def run_single(subj_dir, script, script_args, path_segmanual, path_data, path_da
     })
     # PYTHONNOUSERSITE is originally set in compat/launcher.py. We unset here so that it doesn't affect non-SCT commands
     # in user's pipeline scripts. See: https://github.com/neuropoly/spinalcordtoolbox/issues/3200.
-    del envir['PYTHONNOUSERSITE']
+    envir.pop('PYTHONNOUSERSITE', None)
 
     # Ship the job out, merging stdout/stderr and piping to log file
     try:

--- a/spinalcordtoolbox/scripts/sct_run_batch.py
+++ b/spinalcordtoolbox/scripts/sct_run_batch.py
@@ -196,6 +196,9 @@ def run_single(subj_dir, script, script_args, path_segmanual, path_data, path_da
         'ITK_GLOBAL_DEFAULT_NUMBER_OF_THREADS': str(itk_threads),
         'SCT_PROGRESS_BAR': 'off'
     })
+    # PYTHONNOUSERSITE is originally set in compat/launcher.py. We unset here so that it doesn't affect non-SCT commands
+    # in user's pipeline scripts. See: https://github.com/neuropoly/spinalcordtoolbox/issues/3200.
+    del envir['PYTHONNOUSERSITE']
 
     # Ship the job out, merging stdout/stderr and piping to log file
     try:

